spec/models/zone_spec.rb

```diff
@@ -191,4 +191,15 @@
       ).to eq(1)
     end
   end
+
+  context "validate multi region" do
+    let!(:other_region_id)         { ApplicationRecord.id_in_region(1, ApplicationRecord.my_region_number + 1) }
+    let!(:default_in_other_region) { described_class.create(:name => "default", :description => "Default Zone", :id => other_region_id) }
+    let!(:default_in_my_region)    { described_class.create(:name => "default", :description => "Default Zone") }
+
+    it ".default_zone returns a zone in the current region" do
+      expect(described_class.default_zone).to eq(default_in_my_region)
```

Comment:
Anyone have a suggestion on how to ensure that this test does not result in a false positive since `find_by(:name => "default")` could end up returning the one we want, in the current region without the scoped query

Reply:
No, I think we have to depend on sporadic test failures to find that.

Reply:
It may be possible to create a method/block `in_region(ApplicationRecord.my_region_number + 1) do ; end`
Seems like overkill. I like Brandon's response